source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst

```diff
@@ -1,3 +1,35 @@
 地壳震相
 ========
 
+- **Pg**：在近场，上地壳震源产生的上行 P 波或在上地壳折返的 P 波。也可表示在更大震中距处，
+  整个地壳内部多次 P 波混响产生的震相，群速度约为 5.8 km/s。
+- **Pb**：也叫 P*，下地壳震源产生的上行 P 波或在下地壳折返的 P 波。
+- **PmP**：Moho 不连续面的上界面反射 P 波
+- **Pn**：在上地幔顶部折返的任何 P 波或上地幔顶部震源产生的上行 P 波
+
+- **Sg**：在近场，上地壳震源产生的上行 S 波或在上地壳折返的 S 波。也可表示在更大震中距处，
+  整个地壳内部多次 S 波混响与 SV-P 和/或 P-SV 转换的叠加而产生的震相。
+- **Sb**：也称 S*，下地壳震源产生的上行 S 波或在下地壳折返的 S 波。
+- **SmS**：Moho 不连续面的上界面反射 S 波
+- **Sn**：在上地幔顶部折返的任何 S 波或上地幔顶部震源产生的上行 S 波
+
+- **Lg**：在较大的区域距离处，由整个地壳内部多次 S 波混响与 SV-P 和/或 P-SV 转换的叠加而产生的波群。
+  最大能量的群速度大约是 3.5 km/s。
+- **Rg**： 短周期地壳瑞利波
+
+.. figure:: crustal-phase-raypaths.png
+   :alt: 常见地壳震相的几何路径
+   :width: 60.0%
+   :align: center
```

Comment:
Please optimize the image using https://squoosh.app

Reply:
Use the defaults?

![crustal-phase-raypaths](https://user-images.githubusercontent.com/50591376/103848599-b6373d00-50dd-11eb-90a5-7bb47765f931.png)

Reply:
The default looks good for this image. You can drag the verticle line to left or right, to check the differences between the original image (left side) and the optimized image (right side).

Reply:
How about the format? MozJPEG, browser JPEG or PNG?

Reply:
They are different compression algorithms. Both JPEG and PNG are OK, although PNG is preferred.